packages/xo-web/src/common/xo/index.js

```diff
@@ -3399,6 +3408,8 @@ export const subscribeTunnelState = createSubscription(() => _call('xoa.supportT
 
 export const getApplianceInfo = () => _call('xoa.getApplianceInfo')
 
+export const getApplianceId = () => fetch('./rest/v0/appliance').then(resp => resp.json())
```

Comment:
Does it only get the ID?

Reply:
But we receive an object, right, not a string? If so, we can't call it `getApplianceId`.